pkgs/checks/test/extensions/core_test.dart

```diff
@@ -145,13 +145,20 @@ void main() {
       test('succeeeds for happy case', () {
         checkThat('foo').equals('foo');
       });
+      test('succeeeds for equal empty strings', () {
+        checkThat('').equals('');
+      });
       test('reports extra characters for long string', () {
         checkThat(softCheck<String>('foobar', (c) => c.equals('foo')))
             .isARejection(which: [
           'is too long with unexpected trailing characters:',
           'bar'
         ]);
       });
+      test('reports extra characters for long string against empty', () {
+        checkThat(softCheck<String>('foo', (c) => c.equals(''))).isARejection(
+            which: ['is too long with unexpected trailing characters:', 'foo']);
```

Comment:
Right...but we should word this differently. Like we should say 'is not empty' or something

Reply:
Ah I see what you mean. WDYT now?

Reply:
*chefs kiss* 👍